curriculum/challenges/english/25-front-end-development/workshop-musical-instrument-inventory/6895dffc64e97df0e62e9ef9.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 6895dffc64e97df0e62e9ef9
+title: Step 2
+challengeType: 20
+dashedName: step-2
+---
+
+# --description--
+
+Methods are functions defined inside a class that can perform actions using the class's data. Here is how you create methods:
+
+```python
+def method_name(parameter1, parameter2,...):
+    # Method contents will go here
+```
+
+In Python, the `__init__` method is a special method known as the initializer method, which is called when you create a new instance of the class.
+
+Inside your `MusicalInstrument` class, remove `pass` and add an `__init__` method with three parameters: `self`, `name`, and `instrument_type`. Add `pass` inside the `__init__` method for now.
+
+The `self` parameter refers to the instance of the class and is required as the first parameter for all methods in a class.
```

Comment:
How about this:

```suggestion
The `self` parameter is a standard way to refer the instance of the class and is required as the first parameter for all methods in a class.
```

Reply:
Added missing preposition and a little detail:
```suggestion
The `self` parameter is a standard way to refer to the instance of the class and is required as the first parameter for all instance methods in a class.
```